src/ContentHubClient.php

```diff
@@ -102,7 +102,7 @@ public function ping()
     {
         $config = $this->getConfig();
         // Create a new client because ping is not behind hmac.
-        $client = new Client(['base_uri' => self::makeBaseURL($config['base_uri'])]);
+        $client = new Client(['base_uri' => $config['base_url']]);
         return $this->getResponseJson($client->get('ping'));
```

Comment:
What was wrong with `self::makeBaseURL`?

Reply:
makeBaseURL is there to make sure that base URL will always end with slash and by removing it from the equation, you're compromising it.

Reply:
Fixed